stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -69,6 +71,31 @@ public CardInputView(Context context, AttributeSet attrs, int defStyleAttr) {
         initView(attrs);
     }
 
+    /**
+     * Gets a {@link Card} object from the user input, if all fields are valid. If not, returns
+     * {@code null}.
+     *
+     * @return a valid {@link Card} object based on user input, or {@code null} if any field is
+     * invalid
+     */
+    @Nullable
+    public Card getCard() {
+        String cardNumber = mCardNumberEditText.getCardNumber();
```

Comment:
For the `CardNumberEditText` and `ExpiryDateEditText`, they do have accessors. I wrapped the isValid() calls into the getCardNumber/getValidDate calls (they return null if invalid).